make_rpm

```diff
@@ -185,9 +185,16 @@ mkdir -p usr/local/foglamp
 cd usr/local/foglamp
 (cd ${GIT_ROOT}; 
 if [ -f requirements.sh ]; then
-	sudo ./requirements.sh
+	./requirements.sh
 fi
-rm -rf build; mkdir ./build; cd ./build; cmake ..; make)
+
+if [ -f build.sh ]; then
+	rm -rf build;
+	./build.sh
+else
+	rm -rf build; mkdir ./build; cd ./build; cmake ..; make
+fi)
```

Comment:
As of now we have a single `service` available for FogLAMP. We should place direct steps here for rpm. Likely below
```
source scl_source enable devtoolset-7
rm -rf build; mkdir ./build; cd ./build; cmake ..; make
```

We may move above approach as a generalised way under `foglamp-pkg/service` in **future**. And make service package from there.

Reply:
@ashish-jabble The make_rpm calls build.sh which does the right job depending on CentOS/RHEL 7 or Debian.

That's why you don't see in make_rpm